src/System.Net.Security/src/System/Net/Security/SslAuthenticationOptions.cs

```diff
@@ -36,7 +36,7 @@ internal SslAuthenticationOptions(SslServerAuthenticationOptions sslServerAuthen
             AllowRenegotiation = sslServerAuthenticationOptions.AllowRenegotiation;
             ApplicationProtocols = sslServerAuthenticationOptions.ApplicationProtocols;
             CheckCertName = false;
-            EnabledSslProtocols = sslServerAuthenticationOptions.EnabledSslProtocols;
+            EnabledSslProtocols = FilterOutIncompatibleSslProtocols(sslServerAuthenticationOptions.EnabledSslProtocols);
```

Comment:
When done at this level, does the user have visibility to this filter having kicked in?

Reply:
I don't think so but will double check (I think this would be fine either way)

Reply:
https://source.dot.net/#System.Net.Security/System/Net/Security/SslAuthenticationOptions.cs,9b19271fc222aad5,references says we only pass it in